contrib/ansible/roles/master/tasks/main.yml

```diff
@@ -1,18 +1,42 @@
 ---
+- name: Assures /srv/kubernetes/ dir exists
+  file: path=/srv/kubernetes/ state=directory
+
+- name: Create system kube-cert groups
+  group: name=kube-cert state=present system=yes
+
 - name: write the config file for the api server
   template: src=apiserver.j2 dest=/etc/kubernetes/apiserver
   notify:
-          - restart apiserver
+    - restart apiserver
 
 - name: write the config file for the controller-manager
   copy: src=controller-manager dest=/etc/kubernetes/controller-manager
   notify:
-          - restart controller-manager
+    - restart controller-manager
 
 - name: write the config file for the scheduler
   copy: src=scheduler dest=/etc/kubernetes/scheduler
   notify:
-          - restart scheduler
+    - restart scheduler
+
+- name: Get create cert script from Kubernetes
+  get_url:
+    url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/saltbase/salt/generate-cert/make-cert.sh
+    dest=/srv/kubernetes/make-cert.sh mode=0755 owner=root group=root
```

Comment:
Anything with /srv i'm going to fight against. Since 'atomic' hosts can't create new root directories. We're going to have to work with HOME and /etc/, somehow...